linalg/build.rs

```diff
@@ -153,8 +153,13 @@ fn preprocess_files(
 ) -> Vec<path::PathBuf> {
     let out_dir = path::PathBuf::from(var("OUT_DIR"));
     let mut files = vec![];
-    for f in input.as_ref().read_dir().unwrap() {
-        let f = f.unwrap();
+    let dir_entries = {
+        let mut dir_entries: Vec<fs::DirEntry> =
+            input.as_ref().read_dir().unwrap().map(|f| f.unwrap()).collect();
+        dir_entries.sort_by(|a, b| a.file_name().cmp(&b.file_name()));
```

Comment:
Upon reflection this should probably sort based on the full path, will add commit to do that